infinex-multichain/mainnets/infinex-multichain-blast.toml

```diff
@@ -0,0 +1,80 @@
+name = "infinex-core"
+version = "3.0.0"
+preset = "core"
+description = "Infinex Core Deployments"
+#  pnpm cannon build infinex-multichain/mainnets/infinex-multichain-ethereum.toml --chain-id 81457 --dry-run --upgrade-from
+#  pnpm cannon publish infinex-core-test:3.0.0@core --chain-id 1 --exclude-cloned
```

Comment:
pls update this to use infinex-core and the right chain id

Reply:
done